### ISSUE TITLE
24and crashes on pickblock and place

### ISSUE THREAD

**Issue by Crazy-KSK** [amadornes/SCM-Localization-Issues#209]
https://gist.github.com/12e4670e6ada1967a626576e2ef9a1ef
https://gist.github.com/faa239da80bc1906d6f1b6ded7780049
Just using pickblock and placing the result crashes the circuit.
A blueprint of that circuit also does not do anything when trying to place it.

**Comment by ReTeYeZ** [comment]
Do you happen to have bundled cables running up/down with bundled sticks on the circuit? See here [#190](https://github.com/amadornes/SCM-Localization-Issues/issues/190).

**Comment by Crazy-KSK** [comment]
No, I only use bundled wire on the lowest level all vertical wires are normal redstone not bundled.

I think that I've found the part of my circuit that is crashing, [this](http://i.imgur.com/20Jf8X3.png) is a configuration that I use and it alone crashes when pickblocking and placing it.
